falcon/constants.py

```diff
@@ -186,5 +188,14 @@
 #   None would be ambiguous.
 _UNSET = object()
 
+
+class Missing(Enum):
```

Comment:
IMHO we should be careful with what we expose as part of the public interface; see also #1457.

If we really want to expose this constant, let's document it, and add to our docs. Otherwise, let's prefix it with an underscore.

Reply:
I'll move to typing and underscore it, but will keep `MissingOp` as is.

Once we are done typing stuff we can do a PR to decide which type alias are public and which are not